### DISCUSSION TITLE
afterAssertion() hook is not working.

### DISCUSSION THREAD

**Discussion by AnilCa1905** [webdriverio/webdriverio#12484] [Community]
Hi Team,
I am facing an issue with my wdio test.
I need to use the hook 
afterAssertion: function (params) {
    } to get the value from the params.
    But even with the debugger terminal, this hook is not hitted in any scenario.
    
    Need Help...

**Comment by christian-bromann** [comment]
@AnilCa1905 do you have a reproducible example?

**Comment by AnilCa1905** [comment]
Hi Chris...
Thanks for the replay.
I do have an example
I will share it with you within 30 minutes,
It would be helpful if you can fix this.

Thanks and Regards
Anil

**Comment by AnilCa1905** [comment]
Hi Chris
I cannot send you a reproducible example as the video size is too large.
So as a brief
I am using the afterAssertion() hook inside my Wdio Jasmine framework.
I need to print the params value inside the parameter for the
afterAssertion() hook.
But it is not working in any case.
Please help me if possible.
The exact requirement for me is to print the expect statement I have given
in the withContect() method even if the test is passed or failed.
I am getting the error if the test is passed but iam not getting the
message if the test is passed.
Please help me in this
Regards
Anil

**Reply by christian-bromann** [reply]
A reproducible example is a little Github project I can clone, run a command on and observe the behavior myself. I don't need a video or anything, just a little example project.

**Comment by AnilCa1905** [comment]
Ok, thanks for the clarification.

**Comment by AnilCa1905** [comment]
Hi @christian-bromann 
https://github.com/AnilCa1905/wdio_test
Please find the code 
Thanks

**Reply by erwinheitzman** [reply]
The repo is not accessible

**Reply by AnilCa1905** [reply]
Just a moment.

**Reply by AnilCa1905** [reply]
@erwinheitzman 
Can you check now?
I have added you as a collaborator

**Reply by erwinheitzman** [reply]
Same result

**Reply by erwinheitzman** [reply]
Ah you added me as collaborator, is it so secret you need to have it private? 🤔 would be easier to just make it public as it is just an example right?

**Reply by AnilCa1905** [reply]
Yeah. Done

**Reply by AnilCa1905** [reply]
Can you try now?

**Comment by AnilCa1905** [comment]
The Repo is Public:

https://github.com/AnilCa1905/wdio_test

**Comment by AnilCa1905** [comment]
Hi,
Do you know if we have any solution for the above issue?

**Comment by AnilCa1905** [comment]
@christian-bromann ,@erwinheitzman
Although this issue is not yet resolved...
I am closing this Issue.
Thanks for your time.
Anil
